ee/ucp/admin/configure/route-reflector-deployment.md

```diff
@@ -0,0 +1,216 @@
+# Steps to deploy Route Reflector on UCP 3.0
+
+UCP 3.0 ships with Calico for Kubernetes networking. Calico use BGP as the control plane to distribute Pod routes between the node in the UCP cluster. Calcio BGP is setup as mesh between the nodes. For large scale production-grade deployments it is recommended to deploy Route Reflectors to avoid node to node BGP mesh. This document provides detailed steps to deploy Route Reflectors in UCP cluster.
+
+Note: 
+1) If you are using UCP 3.0 on Azure deploying route reflectors is not required. The networking control plane is handled
+by Azure SDN and not calico.
+
+2) Nodes marked for RR should not run any non system workloads. These nodes will not be gauranteed to work for ingress (swarm) or nodePort(kubernetes)
+
+3) It is recommended to deploy Route Reflectors if the cluster scale exceeds 100 nodes.
+
+4) It is recommended to deploy atleast 2 Route Reflectors in the cluster. Identify the nodes where the route reflector needs to be deployed considering that these nodes will not be available to the scheduler to schedule any workloads. Choose the orchestrator as Kubernetes for these nodes.
+
+
+1)  On one of the nodes in Docker EE cluster, download the calicoctl binary from 
+    ```
+    curl --output /usr/bin/calicoctl -O -L https://github.com/projectcalico/calicoctl/releases/download/v3.1.1/calicoctl --output /usr/bin/calicoctl
+    chmod +x /usr/bin/calicoctl
+    ```
+    
+2) `calicoctl` will need to be configured with the etcd information. There are 2 options:
+
+    a) You can set the environment variables
+    
+       ```
+       export ETCD_ENDPOINTS=127.0.0.1:12379 
+       export ETCD_KEY_FILE=/var/lib/docker/volumes/ucp-node-certs/_data/key.pem 
```

Comment:
@alexmavr Is this the best way to use and configure `calicoctl`? If I have a UCP admin bundle, I can configure `calicoctl` with the keys on my client bundle, and make it point to the UCP cluster, right?

Reply:
That's right, `calicoctl` needs to run directly on the node. We could ask the users to copy their admin client bundles on each node, but re-using the node certs seems more straightforward here